app/src/api/queries.ts

```diff
@@ -44,6 +48,9 @@ async function hentGrunnbeløp() {
 }
 
 export async function hentEksisterendeInntektsmeldinger(uuid: string) {
+  if (uuid === "custom-id") {
```

Comment:
For later: Sleng ut denne custom IDen i en konstant, så det blir lettere å endre på tvers.